docs/v3.x/concepts/services.md

```diff
@@ -343,7 +343,7 @@ module.exports = {
         model: 'restaurant',
         // if you are using a plugin's model you will have to add the `plugin` key (plugin: 'users-permissions')
       });
-      return this.findOne({ id: entry.id });
+      return this.find({ id: entry.id });
```

Comment:
There is no id to pass here it should be a populate option or nothing :)

Reply:
ok updated